test/built-ins/Atomics/wait/posioned-object-for-timeout-throws.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018 Amal Hussein.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-atomics.wait
+description: >
+  False timeout arg should result in a timeout value of 1
```

Comment:
this test throws a TypeError because the timeout object can't find a callable method during the OrdinaryToPrimitive coercion.

Reply:
these were typo's thanks for catching them.